maurofavre-3566/about.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
Hi Mauro, your repository look so great! Excelent work! It seems that you fulfilled all the instructions of the excercises.